src/north/CsvToHttp/CsvToHttp.class.js

```diff
@@ -0,0 +1,241 @@
+const Papa = require('papaparse')
+const fs = require('fs')
+
+const ApiHandler = require('../ApiHandler.class')
+
+/**
+ * Class CsvToHttp - convert a csv file into http request such as POST/PUT/PACTH
+ */
+class CsvToHttp extends ApiHandler {
+  /**
+   * Constructor for CsvToHttp
+   * @constructor
+   * @param {Object} applicationParameters - The application parameters
+   * @param {Engine} engine - The Engine
+   * @return {void}
+   */
+  constructor(applicationParameters, engine) {
+    super(applicationParameters, engine)
+
+    const { applicativeHostUrl, requestMethod, authentication, csvDelimiter, mapping, proxy = null } = applicationParameters.CsvToHttp
+
+    // Creation of an entity request with is composed of all necessaries informations
+    this.request = {}
+    this.request.host = applicativeHostUrl
+    this.request.method = requestMethod
+    this.request.authentication = authentication
+
+    this.mapping = mapping
```

Comment:
mapping does not seem to exist in a new config. You can use this.mapping = mapping || {}

Reply:
or mapping ?? {}